### ISSUE TITLE
Camera movement sluggish

### ISSUE THREAD

**Issue by efossvold** [Try/OpenGothic#693]
Camera keeps moving after mouse movement stops. After I stop moving the mouse to look around there's a gradual slowdown of the camera movement around 2-3 seconds until it finally stops. In the original Gothic II the camera follows mouse movement precisely, when mouse stops moving, camera stops moving. 

Everything else is great, fantastic work! However, this "slowdown delay" (in lack of other words) makes me feel like I'm almost drunk when moving around. I have to constantly compensate for the extra movement of the camera. I don't know if this is just on my computer (macos M1 16GB, Sonoma). Using latest build from CI, but I've experienced it on all previous builds I've tried. Not sure if it's intentional or not, but is there a way to turn off this feature or adjust the camera?

**Comment by Try** [comment]
Hi, @efossvold !

It seems you mean not "movement", but rotation, instead? Then should be duplicate of https://github.com/Try/OpenGothic/issues/612

Also camera is non-trivial topic. In game script camera description has clear indication of rotation speed:
```c++
CLASS CCamSys 
{
	...
	VAR FLOAT veloTrans;	// velocity while easing   to best position
	VAR FLOAT veloRot; 	// velocity while rotating to best orientation
	...
};
```
It's been tested here: https://github.com/Try/OpenGothic/issues/541. Mouse driven-rotation should be high speed, yet limited speed.

**Comment by Nindaleth** [comment]
There are several aspects of the camera reaction to the character rotating):
* camera rotation speed (degrees per second)
* camera rotation momentum (seconds to react to mouse movement start/stop)
* camera rotation fluidity (how many frames seems to pass without camera rotation change)

Initial commenter of #612 talks about camera rotation speed. Then there's a set of camera movement factors (reaction to the character moving) which the 2nd commenter in #612 is possibly talking about, hard to tell.

Maybe it could be useful to have a unique issue per each problem instead of having one generic issue that's harder to discuss?

IMHO this issue is specifically talking about the camera rotation momentum - you've stopped moving the mouse/rotating the character, but the camera continues to rotate afterwards and takes too much time to do it. Which bothers people more sensitive to movement of the camera they can't influence. Do I get this right, @efossvold?

**Comment by efossvold** [comment]
Yes, you are right @Nindaleth, the description of camera rotation momentum is very accurate to what I was initially referring to. I also read #612 before creating this issue and it sounds like something else.

**Comment by efossvold** [comment]
> Hi, @efossvold !
> 
> It seems you mean not "movement", but rotation, instead? Then should be duplicate of #612
> 
> Also camera is non-trivial topic. In game script camera description has clear indication of rotation speed:

Movement vs rotation, not sure how differentiate between these terms. I am referring to when I stop _moving_ the mouse and then perhaps when the camera stops _rotating_, would that be the right term? Anyways, it is not a duplicate of #612. @Nindaleth summed it up well in [his comment](https://github.com/Try/OpenGothic/issues/693#issuecomment-2488176507). From what I understand, it is regarding camera rotation momentum (the time it takes for the camera to stop rotating _after_ mouse movement stops), the sort of "trailing" rotation of the camera following the after mouse movement seizes. Hope that explains it.

**Comment by Try** [comment]
Closing due to: https://github.com/Try/OpenGothic/pull/894
